backend/src/libs/packages/server-application/libs/helpers/get-error-info/get-http-error-info.ts

```diff
@@ -0,0 +1,21 @@
+import { ServerErrorType } from '#libs/enums/enums.js';
+import { type HTTPCode, type HTTPError } from '#libs/packages/http/http.js';
+import { type ValueOf } from '#libs/types/types.js';
+
+import { type ErrorHandler } from './types/types.js';
+
+const getHTTPErrorInfo: ErrorHandler<HTTPError> = (error) => {
```

Comment:
I don't like this convention, but by quality criteria we name abbreviations uppercase only if they the first word and here we have `get` also 🥲 
```suggestion
const getHttpErrorInfo: ErrorHandler<HTTPError> = (error) => {
```

Reply:
I see. I thought it should be the same as the name of the error) Will be changed